src/compiler/crystal/command/docs.cr

```diff
@@ -54,6 +85,7 @@ class Crystal::Command
     included_dirs << File.expand_path("./src")
 
     compiler = Compiler.new
```

Comment:
Is this instantiation of compiler needed? I mean, it is already declarated on line 14

Reply:
Oh, thanks for noticing. That's actually an error, this line needs to go.

Reply:
This ought to go... 🏓

Reply:
Damnit, I overlooked that. Sorry.